src/datastore/pg-store.ts

```diff
@@ -396,6 +396,80 @@ export class PgStore {
     });
   }
 
+  async getMetadataBlocks({ limit, offset }: { limit: number; offset: number }) {
```

Comment:
This function name doesn't make sense. Should just make a plural version of the existing `getBlockWithMetadata`, i.e. `getBlocksWithMetadata`.